packages/coinstac-ui/app/render/components/user/form-login.jsx

```diff
@@ -91,19 +111,12 @@ class FormLogin extends Component {
       <div className={classes.loginFormContainer}>
         <Paper className={classes.paper}>
           <form onSubmit={this.onSubmit}>
-            {
-              auth.error
-              && (
-                <p className={classNames(classes.bottomMargin, classes.error)}>
-                  <strong>Error!</strong> {auth.error}
-                </p>
-              )
-            }
+            {this.renderError()}
```

Comment:
I think that for the sake of readability it would be better like `{ auth.error && this.renderError() }`. This way it's easier to notice that the error message will be rendered only if there's an error.